modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageManagerImpl.java

```diff
@@ -676,17 +676,15 @@ public Cursor<Entry> range(ByteArray keyFrom, @Nullable ByteArray keyTo) throws
      * @see ByteArray
      * @see Entry
      */
-    public @NotNull Cursor<Entry> prefixWithAppliedRevision(@NotNull ByteArray keyPrefix) throws NodeStoppingException {
+    public Cursor<Entry> prefixWithAppliedRevision(ByteArray keyPrefix) throws NodeStoppingException {
         if (!busyLock.enterBusy()) {
```

Comment:
U can use `org.apache.ignite.internal.util.IgniteUtils#inBusyLock(org.apache.ignite.internal.util.IgniteSpinBusyLock, java.util.function.Supplier<T>)`

Reply:
Why should I?

Reply:
This **suggestion** will reduce the amount of similar code.

Reply:
The balance of "force" began to shift =)